pypeit/spectrographs/p200_dbsp.py

```diff
@@ -432,4 +456,22 @@ def config_specific_par(self, scifile, inp_par=None):
         else:
             msgs.error("Your grating " + disp + ' needs a template spectrum for the red arm of DBSP.')
 
+
+        angle = Angle(self.get_meta_value(scifile, 'dispangle'), unit=u.deg).rad
+        slitwidth = self.get_meta_value(scifile, 'slitwid') * u.arcsec
+        lines_mm = float(self.get_meta_value(scifile, 'dispname').split('/')[0]) / u.mm
+
+        theta_m = 35.0 * 2*np.pi / 360. - angle
+        order = 1.
+        platescale = 0.293 * u.arcsec / u.pix
+        pix_size = 15 * u.um
+
+        disp = np.cos(theta_m)/(lines_mm * 12 * u.imperial.inch) * 1e7 * u.AA / u.mm
+        cen_wv = np.abs(1/lines_mm * (np.sin(theta_m) - np.sin(angle)) / order)
+        dlam = slitwidth / platescale * pix_size / u.pix * disp
+
+        resolving_power = cen_wv / dlam
+
+        par['sensfunc']['UVIS']['resolution'] = resolving_power.decompose().value
+
```

Comment:
Nice.  Particularly given that this calculation is done twice, it might be worth generalizing and pulling it out into a single method (maybe in Spectrograph?).  But it's fine if you want to leave it as is for this PR.

Reply:
There are some parts of this calculation that I'm not sure how to generalize to spectrographs with multiple detectors (with different plate scales and/or pixel sizes), and there's some other info (spectrograph focal length) that I'm not sure is stored generally, so I'm gonna punt on this for now.